src/app/containers/Ad/Amp/index.jsx

```diff
@@ -36,29 +63,54 @@ const ampAdPropsDesktop = ({ service }) => ({
   json: JSON.stringify(constructAdJsonData({ service })),
 });
 
+const AMP_ACCESS_DATA = (endpoint) => ({
+  authorization: endpoint,
+  noPingback: true,
+});
+
+const AMP_ACCESS_FETCH = (service) => {
+  const togglesEndpoint = `${process.env.SIMORGH_TOGGLES_URL}/toggles?application=simorgh&service=${service}&geoiplookup=true`;
+
+  return (
+    <script id="amp-access" type="application/json">
+      {JSON.stringify(AMP_ACCESS_DATA(togglesEndpoint))}
+    </script>
+  );
+};
+
 // eslint-disable-next-line react/prop-types
 const AmpAd = ({ service }) => {
   return (
     <>
-      <amp-ad {...ampAdPropsMobile({ service })}>
-        <amp-img
-          placeholder
-          width="320"
-          height="50"
-          src="https://via.placeholder.com/320x50"
-          layout="responsive"
-        />
-      </amp-ad>
-
-      <amp-ad {...ampAdPropsDesktop({ service })}>
-        <amp-img
-          placeholder
-          width="970"
-          height="250"
-          src="https://via.placeholder.com/970x250"
-          layout="responsive"
-        />
-      </amp-ad>
+      <Helmet>
+        {AMP_ACCESS_JS}
+        {AMP_ACCESS_FETCH(service)}
+      </Helmet>
+      <div
+        amp-access="toggles.ads.enabled AND geoIp.advertiseCombined"
```

Comment:
So this will be either true or false depending on the response returned by the endpoint, right?

Reply:
Yep that is correct.

This is an example of the endpoint data response btw: 
```
{
"toggles": {
   "ads": {
      "enabled": true,
      "value": ""
    }
},
"geoIp": {
    "ukCombined": true,
    "advertiseCombined": false,
    "countryCode": "gb"
 }
}
```